src/lib/mutate-processor.ts

```diff
@@ -99,12 +108,13 @@ export async function mutateProcessor(
           errorMessage = "An error occurred with the mutate action.";
         }
 
-        Log.error(actionMetadata, `Action failed: ${errorMessage}`);
+        // Log the alias on failure
+        aliasLogger.error(actionMetadata, `Action failed: ${errorMessage}`);
```

Comment:
Where did we land on discussion whether or not the childLogger should log controller stuff? I think we may want to keep that responsibility on Pepr but it is up for debate.

Reply:
Updated to use Log